results/client.go

```diff
@@ -41,50 +40,70 @@ const (
 	VersionTen     = "v0.10"
 )
 
-// Archive holds an io.ReadCloser that can read through the archive once.
+// Archive holds an io.Reader that can read through the archive once.
```

Comment:
It's more of a ResultsReader then an Archive isn't it?

Reply:
How about ArchiveReader? Mostly I want to avoid `results.Results`

Actually could do `results.Reader`, what do you think of that?